pkg/apis/migration/v1alpha1/resource.go

```diff
@@ -50,23 +41,12 @@ func (r *MigPlan) GetName() string {
 	return r.Name
 }
 
-func (r *MigPlan) GetTouch() string {
-	if r.Annotations == nil {
-		return ""
-	}
-	id, found := r.Annotations[TouchAnnotation]
-	if found {
-		return id
-	}
-
-	return ""
+func (r *MigPlan) MarkReconciled() {
+	r.Status.ObservedGeneration = r.Generation + 1
```

Comment:
This seems to say that we've "observed" as generation past the current one, which doesn't make sense to me. I'm probably missing something, @jortel can you explain?

Reply:
If we write the resource back to the APIserver and generation gets incremented at that point to match the ObservedVersion, then this makes sense to me.